browser/ui/toolbar/brave_app_menu_model.cc

```diff
@@ -54,6 +54,19 @@ void BraveAppMenuModel::InsertBraveMenuItems() {
           IDS_SHOW_BRAVE_SYNC);
     }
   }
+
+  // Insert Create New Profile item
+  auto profile_items_insert_index = GetIndexOfCommandId(IDC_ZOOM_MENU) - 1;
```

Comment:
I probbaly should only have inserted these items if the session isn't a guest session. Right now, the items show as disabled but not hidden. The spec was unclear ;-) I can clean up in https://github.com/brave/brave-browser/issues/5552